test/xpu_api/random/device_tests/common_for_device_tests.h

```diff
@@ -87,7 +87,7 @@ int device_copyable_test(sycl::queue& queue) {
                 Distr device_distr(distr);
                 device_engine.discard(offset);
                 result_type res = device_distr(device_engine);
-                res.store(idx.get_linear_id(), acc.get_pointer());
+                res.store(idx.get_linear_id(), __dpl_sycl::__get_accessor_ptr(acc));
```

Comment:
Using a wrapper from internal API might be a descent solution in short-term. But I would recommend duplicating `__get_accessor_ptr` method in `test/utils_sycl.hpp`.

Reply:
Can we do it properly now? Relying on any implementation detail in tests is bad. We should reduce that. not add new cases.